src/vstart.sh

```diff
@@ -975,8 +989,13 @@ do_rgw()
     RGWSUDO=
     [ $CEPH_RGW_PORT -lt 1024 ] && RGWSUDO=sudo
     n=$(($CEPH_NUM_RGW - 1))
+    #extract rgw_frontends
+    local rgw_fe=`$CEPH_BIN/ceph-conf -c $conf_fn --name $VSTART_SEC rgw_frontends 2>/dev/null`
```

Comment:
by relying on ceph.conf for the frontend type and base port here, that means you'd have to recreate the cluster in order to change either. it seems more flexible to just use the values of `$rgw_frontend` and `$CEPH_RGW_PORT` from the current invocation?